### PR TITLE
Add FeeProvider rake tasks

### PR DESCRIPTION
**Pull request by rantan**
The specificaiton is in README.md

### CODE REVIEW

File: spec/glueby/fee_provider/tasks_spec.rb [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+            expect(tasks).to receive(:status)
+            subject
+          end
+        end
+
+        context 'utxo_pool_size option is specified' do
+          let(:config) { { utxo_pool_size: 10 } }
+
+          it 'creates 10 outputs' do
+            expect(wallet).to receive(:broadcast) do |tx|
```
Comment (azuchi) [inline spec/glueby/fee_provider/tasks_spec.rb:66]: You should check `tx.outputs.count` as well.

File: spec/glueby/fee_provider/tasks_spec.rb [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+            expect(tx.outputs.count).to eq(18)
+            expect(tx.outputs[0...17].map(&:value).uniq).to eq([1_000])
+            # The change output's value should equal to <before amount> - <UTXO pool size> * <fixed fee> - <fee of the tx>
+            expect(tx.outputs[17].value).to eq(100_000_000 - 17 * 1_000 - 1_000)
+          end
+          expect(tasks).to receive(:status)
+          subject
+        end
+      end
+
```
Comment (azuchi) [inline spec/glueby/fee_provider/tasks_spec.rb:149]: Please add the case where there are multiple inputs.

**Review by azuchi — CHANGES_REQUESTED** [review]

File: lib/glueby/fee_provider/tasks.rb [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                .fee(fee_provider.fixed_fee)
+                .build
+        tx = wallet.sign_tx(tx)
+        wallet.broadcast(tx)
+      ensure
+        status
+      end
+
+      def status
+        status = 'Ready'
```
[the remark this answers is not in the document]
Comment (rantan) [reply lib/glueby/fee_provider/tasks.rb]: done

File: lib/glueby/fee_provider/tasks.rb [hunk 4]
```diff
@@ -0,0 +1,121 @@
+module Glueby
+  class FeeProvider
+    class Tasks
+      attr_reader :fee_provider
+
+      def initialize
+        @fee_provider = Glueby::FeeProvider.new
+      end
+
```
[the remark this answers is not in the document]
Comment (rantan) [reply lib/glueby/fee_provider/tasks.rb:18]: done

File: lib/glueby/fee_provider/tasks.rb [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+      def value_to_fill_utxo_pool
+        fee_provider.fixed_fee * (fee_provider.utxo_pool_size + 1) #  +1 is for paying fee
+      end
+
+      def wallet
+        fee_provider.wallet
+      end
+
+      def delimite(num)
```
[the remark this answers is not in the document]
Comment (rantan) [reply lib/glueby/fee_provider/tasks.rb]: done

File: spec/glueby/fee_provider/tasks_spec.rb (hunk 1 above)
Comment (rantan) [reply spec/glueby/fee_provider/tasks_spec.rb:72]: done

File: spec/glueby/fee_provider/tasks_spec.rb (hunk 2 above)
Comment (rantan) [reply spec/glueby/fee_provider/tasks_spec.rb:162]: done

**Review by azuchi — APPROVED** [review]

**Pull request merged**